### DISCUSSION TITLE
Unable to display image when it uploaded to server

### DISCUSSION THREAD

**Discussion by xyzt1402** [deta/cloud-docs#464] [Help]
Hi everyone. I have a problem using server to store images. The file i upload to server and the file i downloaded is different. The one that i downloaded is unable to open. Maybe the server has the problem of encode and decode img file. I followed the instruction from deta document of upload and download with nodejs.
![image](https://user-images.githubusercontent.com/75368538/188668464-bb1117c7-f609-41f7-a98d-49761057108a.png)
![image](https://user-images.githubusercontent.com/75368538/188668504-48e7c3d1-cc5d-4dc9-aae5-727d58e3a7ac.png)
![image](https://user-images.githubusercontent.com/75368538/188668752-b72c675a-19fc-427e-b22f-89cf0de884a1.png)

**Comment by abdelhai** [comment]
hi
1. please share the responsible code so we can see what's wrong
2. make sure you test that it is working locally before you deploy

**Comment by nghiepdev** [comment]
Update document
https://docs.deta.sh/docs/common_issues#nodejs-micros-cannot-serve-binary-files
